src/ashley/api/serializers.py

```diff
@@ -0,0 +1,45 @@
+"""Serializer for ashley api."""
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+from ashley.context_mixins import get_current_lti_session
+from ashley.defaults import (
+    _FORUM_ROLE_INSTRUCTOR,
+    _FORUM_ROLE_MODERATOR,
+    _FORUM_ROLE_STUDENT,
+)
+
+User = get_user_model()
+
+
+class UserSerializer(serializers.ModelSerializer):
+    """Serializer for User model."""
+
+    role = serializers.SerializerMethodField()
+
+    class Meta:
+        model = User
+        fields = ["id", "public_username", "role"]
+        read_only_fields = ["public_username", "role"]
+
+    def get_role(self, obj):
+        """Describes the role of the user"""
+        context = get_current_lti_session(self.context.get("request"))
```

Comment:
You should rename this variable because it is confusing with the serializer `context`